web.jsf12/external/jsf-api-1.2_05-license.txt

```diff
@@ -313,7 +313,16 @@ Participant Software against such Participant either
 unilaterally or pursuant to a written agreement with
 Participant.
 
-6.3. In the event of termination under Sections 6.1 or 6.2
+6.3. If You assert a patent infringement claim against
+Participant alleging that the Participant Software directly
+or indirectly infringes any patent where such claim is
+resolved (such as by license or settlement) prior to the
+initiation of patent infringement litigation, then the
+reasonable value of the licenses granted by such Participant
+under Sections 2.1 or 2.2 shall be taken into account in
+determining the amount or value of any payment or license.
+
+6.4. In the event of termination under Sections 6.1 or 6.2
```

Comment:
This is not CDDL-1.0, but CDDL-1.1  -- the CDDL-1.0 file in `nbbuild/licenses/CDDL-1.0` is not correct. See my changeset here: https://github.com/apache/incubator-netbeans/pull/807/commits/ba70c38b946d16231082d3b861acfe6aa9606117 

Please wait till that PR is merged, so that we ca correctly handle this.